artifactory_test.go

```diff
@@ -207,6 +207,27 @@ func TestArtifactoryUploadPathWithSpecialCharsAsNoRegex(t *testing.T) {
 	cleanArtifactoryTest()
 }
 
+func TestArtifactoryIgnoreChecksumOnFilteredResource(t *testing.T) {
+	initArtifactoryTest(t, "")
+	filePath := "testdata/filtered/a.txt"
+
+	// Upload a filtered resource
+	runRt(t, "upload", filePath, tests.RtRepo1, "--flat", "--target-props", "artifactory.filtered=true")
+	searchFilePath, err := tests.CreateSpec(tests.SearchAllRepo1)
+	assert.NoError(t, err)
+
+	inttestutils.VerifyExistInArtifactory(tests.GetSimpleUploadFilteredRepo1(), searchFilePath, serverDetails, t)
+
```

Comment:
Let's disable logging to prevent "Error"s in the logs
```suggestion
	// Discard output logging to prevent negative logs
	previousLogger := tests.RedirectLogOutputToNil()
	defer log.SetLogger(previousLogger)
```